### PR TITLE
volume_type changed to gp3

### PR DESCRIPTION
**Pull request by hactiv8r**
# Description

volume_type changed to gp3. 

Fixes 

# Checklist:

- [ ] Update the changelog with any notable changes such as features added, changed, fixed, and/or removed.
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -89,7 +89,7 @@ POLICY
   }
   ebs_options {
     ebs_enabled = "true"
-    volume_type = "gp2"
+    volume_type = var.ebs_type =! null ? var.ebs_type : "gp3"
```
Comment (ec209) [inline main.tf:92]: Can you actually specify the gp3/gp2 volume type?

Comment (ec209) [reply main.tf:92]: my question is more like, does AWS actually let you choose the volume type? 
I cannot really find the option on AWS console so.

Comment (mattbrescia) [reply main.tf]: Looking at all the documentation seems that we can only choose between gp2 and io1 actually... I need to test it to have 100% sure it doesn't work with gp3 but io2 doesn't support Elastic Volumes operations, so I 100% don't know.
I asked Yusuf to do a change to accept the env and sets the default as gp2 so we can merge without fear

**Review by mattbrescia — APPROVED** [review]

File: main.tf [hunk 2]
```diff
@@ -89,7 +89,7 @@ POLICY
   }
   ebs_options {
     ebs_enabled = "true"
-    volume_type = "gp2"
+    volume_type = var.ebs_type =! null ? var.ebs_type : "gp2"
```
Comment (ec209) [inline main.tf:92]: actually this would been just 
`var.ebs_type`
since the default value is already in ebs_type variable definition

**Comment by hactiv8r** [comment]
@ec209 Thank you for reviewing, change made as you advised.

**Pull request merged**